frontend/components/UI/Form.js

```diff
@@ -0,0 +1,9 @@
+import classes from "../../styles/Form.module.css";
```

Comment:
Not for this PR but when refactoring/cleaning up the code, what do you think about colocating the css module files with the components, example...
```
- frontend
   - components
      - UI
         - Form.js
         - Form.module.css
```
This will make it easier to navigate to the css module files to make changes

Reply:
It makes sense, at the beginning it was simpler to manage the styles that way but as they increased everything got more complex haha